### PR TITLE
Improve expected results ci checks

### PR DESCRIPTION
**Pull request by nlisgo**
Also upgrading the api-sdk and api-dummy to support version 5 of article-vor

### CODE REVIEW

**Comment by nlisgo** [comment]
Holding off from merging until end2end cleanup is fixed: https://alfred.elifesciences.org/job/process/job/process-clean-end2end/

**Comment by lsh-0** [comment]
> Holding off from merging until end2end cleanup is fixed: https://alfred.elifesciences.org/job/process/job/process-clean-end2end/

fixed

File: bin/ci-expected-results [hunk 1]
```diff
@@ -1,17 +1,54 @@
...
+    if [ "$actual_total" -gt "$expected_total" ]; then
         echo "(have more test fixtures been added to api-dummy?)"
     fi
     exit 2
 fi
+
+echo "CHECK: Search for article id"
+actual_first_id=$(curl -v "localhost/search?for=04395&page=1&per-page=10&sort=relevance&order=desc&use-date=default" | jq ".items[0].id")
+expected_first_id='"04395"'
+if [[ $actual_first_id != $expected_first_id ]]; then
```
Comment (lsh-0) [inline bin/ci-expected-results:21]: I would prefer these:

`if [[ $actual_first_id != $expected_first_id ]]; then` 

to look like this:

`if [ "$actual_first_id" != "$expected_first_id" ]; then`

double brackets are for extended types of comparisons. They're a little bit cleverer with slightly different parsing rules and in the cases used in this changeset they're unnecessary.

Comment (lsh-0) [reply bin/ci-expected-results:21]: PR here: https://github.com/elifesciences/search/pull/306

if you approve then the changes will be merged into this PR

**Review by lsh-0 — APPROVED** [review]
I've made some suggestions but none of them are blockers.

**Pull request merged**